components/ShowCharacter.js

```diff
@@ -17,6 +19,16 @@ export default function ShowCharacter() {
               <br />
               <li>{character.information}</li>
             </ul>
+            <div>
+              <button
+                type="button"
+                onClick={() => {
+                  deleteCharacter(character.id);
+                }}
+              >
+                <img src={'/trash-can-outline.svg'} width="20px" />
+              </button>
+            </div>
```

Comment:
Do you really need the div here? It doesn't group elements together nor serves any other purpose than just applying padding. I am sure you can find a styling solution with less elements and css properties ;)

Reply:
The div is there for when the edit button is added, that these are individually styleable :)
So that they are not affected by the flexbox around the outside